src/routes/userRoutes/logout.js

```diff
@@ -0,0 +1,11 @@
+const express = require("express");
```

Comment:
I prefer having all my route declaration in one file under `user.js` or `auth.js`. Creating a new router for each sub route of a sub route creates unnecessary overhead.

You could have a single `userRoutes.js` or `authRoutes.js` which has the declarations for all related routes in itself rather than abstracting it away to different files.